DbTables/inc/SimEfficiencies.hh

```diff
@@ -41,8 +41,7 @@ namespace mu2e {
     std::vector<Row> const& rows() const {return _rows;}
     std::size_t nrow() const override { return _rows.size(); };
     //    virtual std::size_t nrowFix() const { return 3; };
-    size_t size() const override { return _csv.capacity() +
-        + nrow()*nrow()/2 + nrow()*sizeof(Row); };
+    size_t size() const override { return _csv.capacity(); };
```

Comment:
Sorry, I meant to remove only the nrow*nrow part, so please restore  this: nrow()*sizeof(Row)

Reply:
we have not adopted policy on resolving converstations.  I've used it to reduce the clutter on complicated feedback once an issue has been resolved, but of course that can be misleading if not everyone agrees it has been resolved.  For now I suggest either party may resolve a conversation if it's clear the issue is resolved.

Reply:
Whoops. I should have thought more about it. Added back.

Reply:
It looks like you added it back on the 16th, but now it is gone again.

Reply:
I think it is still there (see https://github.com/Mu2e/Offline/pull/265/files#diff-f035c545608fa2fabc2ea0f195903e43bad3ba793f36f5a22358cb651345e015). It's just this comment thread attaches to a specific commit...

Reply:
Clearly I have not mastered the review views..